wire-runtime/src/main/java/com/squareup/wire/ProtoReader.java

```diff
@@ -174,11 +178,10 @@ private void skip(long count) throws IOException {
     source.skip(count);
   }
 
-  /** Reads a {@code string} field value from the stream. */
-  String readString() throws IOException {
-    int count = readVarint32();
-    pos += count;
-    return source.readUtf8(count);
+  int readByte() throws IOException {
+    source.require(1); // Throws EOFException if insufficient bytes are available.
+    pos++;
+    return source.readByte() & 0xff;
```

Comment:
Cool. Later we should make those claims! (Currently it calls `require()` internally anyway).